### ISSUE TITLE
[1][Style-Guide] Tooltip Layout

### ISSUE THREAD

**Issue by startailcoon** [bitshares/bitshares-ui#2293]
The Ant Design Tooltip needs to be changed to reflect a better choice of font, font-size and design.

The background of the tooltip is floating into the background of the application. It would need a border to single it out better.

![bitshares-tooltip](https://user-images.githubusercontent.com/12114550/49696131-7b344980-fba5-11e8-958c-8be01b6e928d.gif)

**Comment by sschiessl-bcp** [comment]
Do you have github issue preview on?
![image](https://user-images.githubusercontent.com/33128181/49716033-bfc9ee80-fc52-11e8-9675-87fa687d651f.png)

**Comment by sschiessl-bcp** [comment]
Anyways, I closed my other issue as a duplicate
#2229

**Comment by startailcoon** [comment]
@sschiessl-bcp yes, but I'm still unused to having it and must have missed that one.

**Comment by startailcoon** [comment]
@gibbsfromncis are you working on this?

**Comment by gibbsfromncis** [comment]
@startailcoon Nope. I'll work on it before end of this sprint

**Comment by startailcoon** [comment]
@gibbsfromncis do you have a progress update?

**Comment by gibbsfromncis** [comment]
@startailcoon will work on it today/tomorrow

**Comment by sschiessl-bcp** [comment]
> @startailcoon will work on it today/tomorrow

Happened?

**Comment by gibbsfromncis** [comment]
@startailcoon @sschiessl-bcp 

![tooltip](https://user-images.githubusercontent.com/35899973/57514200-905e0180-7318-11e9-93f3-c8fc8653286d.gif)

I've looked more deep on this issue and my position is:
- I see no reason to change font, size or color. It is OK. I suppose @startailcoon you have something wrong (or had?) with your zoom.
- I've updated box-shadow for tooltip container and made it more stronger (check the licecap)
